### PR TITLE
minor: remove useless `.get()`

### PR DESCRIPTION
**Pull request by jackwener**
# Which issue does this PR close?

N/A

# Rationale for this change

# What changes are included in this PR?

# Are these changes tested?

# Are there any user-facing changes?

### CODE REVIEW

File: datafusion/optimizer/src/propagate_empty_relation.rs [hunk 1]
```diff
@@ -149,14 +149,11 @@ fn binary_plan_children_is_empty(plan: &LogicalPlan) -> Result<(bool, bool)> {
     // all binary-plan need to deal with separately.
     match inputs.len() {
         2 => {
-            let left = inputs.get(0).unwrap();
-            let right = inputs.get(1).unwrap();
-
-            let left_empty = match left {
+            let left_empty = match inputs[0] {
```
Comment (Dandandan) [inline datafusion/optimizer/src/propagate_empty_relation.rs:152]: We could also match on `inputs` directly I think.

Comment (Dandandan) [reply datafusion/optimizer/src/propagate_empty_relation.rs:152]: Instead of on the length

Comment (jackwener) [reply datafusion/optimizer/src/propagate_empty_relation.rs]: thanks @Dandandan

**Review by metesynnada — COMMENTED** [review]
LGTM!

**Review by Dandandan — APPROVED** [review]

**Comment by Dandandan** [comment]
Thanks @jackwener

**Pull request merged**